azavea/temperate
# WIP: Upgrade Angular

## Overview: WIP DO NOT MERGE

Companion to:
 - azavea/climate-change-components#52
 - azavea/ngx-bootstrap#1


### Demo

`yarn build` succeeds:
![image](https://user-images.githubusercontent.com/960264/60130350-0c939380-9765-11e9-88fa-b459c23a85bd.png)

`yarn build:prod` fails with AOT error:
![image](https://user-images.githubusercontent.com/960264/60130391-203efa00-9765-11e9-885f-e33a1510fd45.png)


## Testing Instructions

 * Check out [components branch](https://github.com/flibbertigibbet/climate-change-components/tree/feature/kak/angular-update)
 * `yarn build && yarn pack` components and copy to `planit` directory
 * Check out [ngx-bootstrap branch](https://github.com/flibbertigibbet/ngx-bootstrap/tree/feature/kak/upgrade-angular)
 * `yarn build && yarn pack` and copy to `angular` directory (above `planit`)
 * `yarn build` in `planit` directory should succeed
 * `yarn build:prod` should produce error above


Connects #263
